<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="QA-Scheduler" failures="0" tests="8" name="com.associatePortal.Associate_Login" time="24.990" errors="1" timestamp="13 Nov 2018 05:18:48 GMT">
  <testcase classname="com.associatePortal.Associate_Login" name="TCBA01006" time="3.403"/>
  <testcase classname="com.associatePortal.Associate_Login" name="TCBA01003" time="0.359"/>
  <testcase classname="com.associatePortal.Associate_Login" name="TCBA01002" time="0.356"/>
  <testcase classname="com.associatePortal.Associate_Login" name="TCBA01004" time="2.406"/>
  <testcase classname="com.associatePortal.Associate_Login" name="TCBA01001" time="0.609"/>
  <testcase classname="com.associatePortal.Associate_Login" name="TCBA01008" time="4.294"/>
  <testcase classname="com.associatePortal.Associate_Login" name="TCBA01005" time="3.639"/>
  <testcase classname="com.associatePortal.Associate_Login" name="TCBA01007" time="9.924">
    <error message="expected [You have been locked out. Please contact support.] but found [Invalid username or password.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [You have been locked out. Please contact support.] but found [Invalid username or password.]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at com.associatePortal.Associate_Login.TCBA01007(Associate_Login.java:221)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- TCBA01007 -->
</testsuite> <!-- com.associatePortal.Associate_Login -->
